.github/workflows/test.yml

```diff
@@ -9,6 +9,7 @@ on:
     paths-ignore:
       - '**.md'
   pull_request:
+
   # Allow manually triggering the workflow.
```

Comment:
Not sure why an extra new line is being added ? `workflow_dispatch` is just another workflow trigger, just like `pull_request`, so this is not a different "section" or anything in the workflow.

(same remark for the other workflow)

Reply:
I validated this against a schema for GitHub Workflows, and it didn't like the comment on the next line after `pull_request:` (presumably wanted an indent); adding a clear line space made the validation happy again.